### PR TITLE
added monitor plugin check-netif.rb

### PR DESCRIPTION
**Pull request by autumnw**
Based on netif-metics.rb, provided the monitor plugin for netif.

### CODE REVIEW

File: bin/check-netif.rb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+         default: 'eth0'
+
+  option :critical,
+         description: 'Critical throughput in KB',
+         short: '-c rx_kb_per_sec,tx_kb_per_sec',
+         long: '--critical rx_kb_per_sec,tx_kb_per_sec',
+         default: '1048576,1048576'
+
+  option :warning,
+         description: 'Waring throughput in KB',
```
Comment (analytically) [inline bin/check-netif.rb]: Warning

File: bin/check-netif.rb [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+         description: 'Comma delimited list of interface names',
+         short: '-i INTERFACES',
+         long: '--interfaces INTERFACES',
+         default: 'eth0'
+
+  option :critical,
+         description: 'Critical throughput in KB',
+         short: '-c rx_kb_per_sec,tx_kb_per_sec',
+         long: '--critical rx_kb_per_sec,tx_kb_per_sec',
+         default: '1048576,1048576'
```
Comment (analytically) [inline bin/check-netif.rb]: What's this default based upon?

**Comment by analytically** [comment]
Can you rename this check to check-netif-throughput.rb?

File: bin/check-netif.rb [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+require 'socket'
+
+#
+# Netif Monitoring
+#
+class NetIFMetrics < Sensu::Plugin::Check::CLI
```
Comment (analytically) [inline bin/check-netif.rb]: Bad class name.

**Comment by eheydrick** [comment]
@autumnw Bump for feedback.

**Comment by majormoses** [comment]
@autumnw are you planning on continuing the work here or should I close this?

**Comment by majormoses** [comment]
Feel free to re-open if you want to pursue but its been dead for a very long time.

**Pull request closed without merging**